scripts/junos/config_tables/InterfaceConfigTables.yml

```diff
@@ -0,0 +1,24 @@
+# To understand the following definitions, refer to
```

Comment:
Please add a description on top of these `.yml` files so that it's easy to follow why it's there in the first place. 

This looks very similar to `ConfigTables.yml`. And idk where you are using this.